src/sentry/search/events/fields.py

```diff
@@ -2841,6 +2917,46 @@ def _resolve_user_misery_function(self, args: Mapping[str, str], alias: str) ->
             alias,
         )
 
+    def _resolve_percentile(
+        self,
+        args: Mapping[str, Union[str, Column, SelectType, int, float]],
+        alias: str,
+        fixed_percentile: float = None,
+    ) -> SelectType:
+        return Function(
```

Comment:
This will return slightly different results for p100, which is actually implemented with `max` instead. I believe this is because the quantile function is non-deterministic (especially on larger datasets) 
https://clickhouse.tech/docs/en/sql-reference/aggregate-functions/reference/quantile/

Reply:
👍  added a condition to use `max` when querying for p100